service/src/main/resources/db/migration/V001__Schema.sql

```diff
@@ -0,0 +1,170 @@
+CREATE TABLE qrtz_job_details
```

Comment:
How we run this in the pipeline?

Reply:
yes, we do, flyway is used to do db migration